js/index.js

```diff
@@ -0,0 +1,34 @@
+$(function(){
+
+	//declaring a variable
+	var textInput;
+
+	$("#submit-btn").click(changeCity);
+	
+	//When clicking prevet default
+	function changeCity(event){
```

Comment:
Everything works Great. Awesome!

:hammer: The only issue is the indentation. These projects will serve as reference material for you later on to look back on. I still look back on my projects from GA too. Without proper indentation, the code becomes really difficult to read. We can clean it up together if you'd like. But just fix the indentation and this HW will be complete :)